build/assets/tuned/openshift-node-performance

```diff
@@ -1,34 +1,49 @@
 [main]
-summary=Openshift node optimized for deterministic performance at the cost of increased power consumption, focused on low latency network performance
-include=openshift-node
+summary=Openshift node optimized for deterministic performance at the cost of increased power consumption, focused on low latency network performance. Based on Tuned 2.11 and Cluster node tuning (oc 4.5)
+include=openshift-node,cpu-partitioning
 
-[cpu]
+# Inheritance of base profiles legend:
+# cpu-partitioning -> network-latency -> latency-performance
```

Comment:
well this is the way tuned is built (and as a consequence NTO) this is not something new.
I added this legend for better transparency rather than complicating things. 
I will write a how-to guide in a later PR for operator level (medium-advanced) on ways to change settings